src/app/services/http.service.ts

```diff
@@ -109,6 +109,14 @@ export class HttpService {
     );
   }
 
+  getAllHistoryRewards(start?: Date, end?: Date): Observable<RewardDto[]> {
+    start = start || new Date(0);
+    end = end || (new Date());
+    return this.http.get<RewardDto[]>(`${environment.apiEndpoint}/api/transactions/history/reward/?start=${(start.getTime()/1000).toFixed(0)}&end=${(end.getTime()/1000).toFixed(0)}`).pipe(
```

Comment:
please, use number instead of Date in arguments

Reply:
👍